server/src/main/java/io/eiren/math/Vector3d.java

```diff
@@ -1,6 +1,7 @@
 package io.eiren.math;
 
-import com.jme3.math.Vector3f;
+
+import io.github.axisangles.ktmath.Vector3;
 
 
 public class Vector3d implements Cloneable {
```

Comment:
Is this used? We can probably remove the class.

Reply:
Done